iOS/DittoPOS.xcodeproj/project.pbxproj

```diff
@@ -560,6 +566,7 @@
 				DEVELOPMENT_ASSET_PATHS = "\"DittoPOS/Preview Content\"";
 				DEVELOPMENT_TEAM = 3T2VMFZPPQ;
 				ENABLE_PREVIEWS = YES;
+				ENABLE_USER_SCRIPT_SANDBOXING = NO;
```

Comment:
Without enabling this, the build phase that copies things from `.env` to `Env.swift` would always fail for me in Xcode 15.3